components/states/DiveShopConfigFetchState.jsx

```diff
@@ -20,6 +20,9 @@ const DiveShopConfigFetchState = () => {
     if (url_hash) {
       fetchData(url_hash);
     }
+    send({
+      type: STATE_ACTIONS.FETCH_SUCCESS,
+    });
```

Comment:
nope, should not be sent here!

Reply:
Did it so I can test locally - forgot to remove